examples/advanced/example_model_optimization.py

```diff
@@ -302,31 +304,14 @@ def train(
 ########################################################################################
 # Depending on the dataset and your setup the training can take a couple of hours. To
 # avoid this, we provide transformer weights that were trained with the scheme above.
-#
-# .. note::
-#
-#   If you want to perform the training yourself, set
-#   ``use_pretrained_transformer=False``. If you do, you also need to replace
-#   ``dataset = None`` below with the dataset you want to train on.
-#
-# .. note::
-#
-#   The weights of the provided transformer were trained with the
-#   `2014 training images <http://images.cocodataset.org/zips/train2014.zip>`_ of the
-#   `COCO dataset <https://cocodataset.org/>`_. The training was performed for
-#   ``num_epochs=2`` and ``batch_size=4``. Each image was center-cropped to
-#   ``256 x 256`` pixels.
 
-use_pretrained_transformer = True
-checkpoint = "example_transformer.pth"
 
-if use_pretrained_transformer:
+def download_example_transformer_weights():
+    checkpoint = "example_transformer.pth"
     if path.exists(checkpoint):
-        state_dict = torch.load(checkpoint)
+        return torch.load(checkpoint)
     else:
-        # Unfortunately, torch.hub.load_state_dict_from_url has no option to disable
-        # printing the downloading process. Since this would clutter the output, we
-        # suppress it completely.
+
```

Comment:
There is no need for an `else` here since `if` branch `return`'s. Please remove the `else` and "unindent" contents by one level.